graph/src/components/store.rs

```diff
@@ -1344,13 +1354,13 @@ pub trait StatusStore: Send + Sync + 'static {
 /// `EntityOperation`, we already know whether a `Set` should be an `Insert`
 /// or `Update`
 #[derive(Clone, Debug, PartialEq, Eq)]
-pub enum EntityModification {
+pub enum EntityModification<K = EntityKey> {
```

Comment:
What's the reason to not use a full `EntityKey` for this PR? I think genericizing this actually makes the code less clear.

Reply:
Lots of unnecessary data redundancy, mainly. We don't need to know `EntityKey::subgraph_id` for every entity because it's an invariant.

By the way, now that I'm thinking about it, it would be best to actually use `EntityOperation` rather than `EntityModification`. Still, the discussion above stands still.

Reply:
Using `EntityOperation` + `EntityKey` now. Marking this as resolved.